interfaces/inc/c_pal/threadpool.h

```diff
@@ -21,10 +21,13 @@ extern "C" {
 #endif
 
 typedef struct THREADPOOL_TAG THREADPOOL;
-typedef struct TIMER_INSTANCE_TAG* TIMER_INSTANCE_HANDLE;
+typedef struct TIMER_INSTANCE_TAG TIMER_INSTANCE;
+typedef struct TIMER_INSTANCE_TAG* TIMER_INSTANCE_HANDLE; //todo: if this one is needed
```

Comment:
should probably remove TIMER_INSTANCE_HANDLE

Reply:
removed

Reply:
Not removed, I can still see it.